mod/filter/set/abstract/01_filter_formgroups.rb

```diff
@@ -30,8 +38,12 @@
     multiselect_filter :metric_type
   end
 
-  view :filter_metric_value_formgroup, cache: :never do
-    select_filter :metric_value, "exists"
+  view :filter_status_formgroup, cache: :never do
+    select_filter :status, "exists"
+  end
+
+  view :filter_updated_formgroup, cache: :never do
+    select_filter :updated
```

Comment:
what was formerly the "answer" filter is now divided into status and updated.  I haven't added outliers back in yet.  (before it was appearing on company pages where it wasn't functional)